pnpm-lock.yaml

```diff
@@ -28252,7 +28252,7 @@ packages:
     resolution: {integrity: sha512-m7Lllj9n/S6sOkCkRftpM7L24uvmfXQFedlW/4hENcuJH1HHm9u5EgxZb9uVjQSCGrbBWBkOGgcTxNg36r6ywA==}
     dependencies:
       fast-glob: 3.3.2
-      minimatch: 9.0.3
+      minimatch: 9.0.4
```

Comment:
The changes in this file seem unrelated to version bump.

Reply:
this change is deduplicating 9.0.3 with 9.0.4 (which we already had a copy of included). No idea why this would happen in this change, but seems good and is likely related to the resolution step that is caused during pnpm install due to having file dependencies (which I think just got fixed on main).

Regardless I don't know whats up with the `requiresBuild: true` lines. Liley best to just revert the changes to this file.

Reply:
reverted changes from pnpm-lock.yaml file